apps/console/src/routes.tsx

```diff
@@ -240,7 +240,7 @@ export const getAppViewRoutes = (): RouteInterface[] => {
                     showOnSidePanel: false
                 },
                 {
-                    component: lazy(() => import("../../admin.applications.v1/pages/applications-settings")),
+                    component: lazy(() => import("../../../features/admin.applications.v1/pages/applications-settings")),
```

Comment:
Let's use scoped imports here.
```suggestion
                    component: lazy(() => import("@wso2is/admin.applications.v1/pages/applications-settings")),
```

Reply:
Fixed. Thanks.